plugins/metrics/parser/src/metricsparser.cpp

```diff
@@ -51,6 +51,40 @@ std::vector<std::string> MetricsParser::getDependentParsers() const
   return std::vector<std::string>{};
 }
 
+bool MetricsParser::preparse()
+{
+  if(!_fileIdCache.empty())
```

Comment:
space

Reply:
Fixed in 64cb407581662e78675baf916a95dba2b765b3e4.